### ISSUE TITLE
Cant open MVT for writing

### ISSUE THREAD

**Issue by MangoChris** [contra/node-gdal-next#43]
Package Version: 2.6.0
Node Version: 14.9.0

Command:

var data = gdal.open('/tmp', 'w', 'MVT');

Expected behaviour:

data variable contains Dataset

Actual behaviour:

Uncaught Error: ERROR 6: Only vector creation supported
Uncaught Error: Error creating dataset

Notes:

All other file formats 'Esri Shapefile', 'KML' etc work as expected.

**Comment by mmomtchev** [comment]
@MangoChris are you using `gdal-next` with your own shared GDAL library? By building with `--shared_gdal`? MVT write support is not working with the prebuilt binaries
If so, try with
```js
var data = gdal.open('/tmp', 'w', 'MVT', 0, 0, 0, 'Unknown');
```

The default options (datatype `GDT_Byte`) fool the MVT driver that you want to create a raster dataset which it does not support

**Comment by contra** [comment]
Published as 2.6.1 - thanks @mmomtchev!

**Comment by MangoChris** [comment]
Thanks for the quick response guys, adding 'Unknown' did the trick!
